### DISCUSSION TITLE
Please help!!! "Filtered Folder Collections" query is too intensive for GitHub. Need a workaround! Currently exhausted!

### DISCUSSION THREAD

**Discussion by JG515** [decaporg/decap-cms#6827] [Q&A]
Hello,

We have implemented a relation widget that uses a filtered folder collection ( https://decapcms.org/docs/collection-types/#filtered-folder-collections ) to aggregate collections of multiple types together so they can be selected from the drop down box in the CMS.

This works well on our local development environment, but when we put it up on our server environment which feeds our files from GitHub, the server support staff say that the query is too intense for GitHub to allow the drop down box to populate.

We have been trying every other way we can possibly think of to get multiple collection types to populate our drop downs. We have a lot of them on the site we are trying to migrate from WordPress into Decap so this is central to the way our site works and the data model is set up.

Can someone please help us implement a workaround or other method. I am absolutely exhausted at this point.

We have tried using adding collections with the .addCollection method in the eleventy.js file. We can print the created collection, but Decap won't use it to populate the relation widget.

We have tried using "Referencing a file collection list field " ( https://decapcms.org/docs/widgets/relation/ ) but we can't seem to get that to work because there isn't enough explanation. I can't tell if we need two collections defined in the admin/config.yml file, or what the yml file is supposed to look like, or if I'm putting the files in the right directories, or if this is even the right way to make the relation widget drop down box use a yaml or json file to populate its values.

I guess that is basically what we need: how do we make the relation widget drop down box use a yaml or json file to populate its values?

Thanks so much! Fingers crossed that someone can help!

**Comment by martinjagodic** [comment]
@JG515 Yes, this is a limitation of Github. How many requests do you need? There is a Github Enterprise plan, which gets you 15000 requests per hour rather than 5000.

For more detailed help please add more details about the project and your config.yaml.
